server/models/Profile.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+const reqString = {
+    type: String,
+    require: true
+};
+
+const ProfileSchema = new mongoose.Schema({
+    userId: {
+        type: mongoose.Schema.Types.ObjectId,
+        require: true,
+        ref: 'user'
+    },
+    firstName: reqString,
+    lastName: reqString,
+    email: reqString,
+    description: {
+        type: String,
+        maxlength: 300
+    },
+    gender: String,
+    birthDate: Date,
+    phoneNumber: [String],
+    hourlyRate: Number,
+    address: {
+      city: String,
+      province: String,
+      country: String,
+    },
+    isAvailable:{
+        type: Boolean,
+        default: false
+    },
+    availability: {
+        days: [String],
+        hours: String
+    },
```

Comment:
Can you expound on your scheme for storing availability? Are these days of the week and number of hours per day? The hours may have to be split to start and end times.

Reply:
No these days are not the day of the week. The days variable store the specific dates that selected by the user from the calendar.

Reply:
Great! You can use `dates` instead of `days` and [Date] instead of [String]. You can improve this field by making your availability scheme more self-explanatory for others who would be using this model.

Reply:
We will talk more about this @lesurafel on our meeting tomorrow on the best approach here for a better UI/UX. 😄